xbmc/cores/AudioEngine/Utils/AEChannelInfo.h

```diff
@@ -32,7 +32,7 @@ class CAEChannelInfo {
 
 public:
   CAEChannelInfo();
-  CAEChannelInfo(const enum AEChannel* rhs);
+  explicit CAEChannelInfo(const enum AEChannel* rhs);
```

Comment:
if you ask me, you will always get the explicit code. i do not like implicit casts one bit.